src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
 // write your code here
+const rows = document.querySelector('tr');
+
+rows.forEach((row) => {
```

Comment:
Since `rows` is not an array or NodeList (due to the use of `querySelector`), the `forEach` method will not work. Ensure `rows` is a NodeList by using `querySelectorAll` as suggested above.